config/initializers/1_rack_query_parser.rb

```diff
@@ -0,0 +1,8 @@
+require 'rack/request'
+require 'query_parser'
+
+Rack::Request::Env.class_eval do
```

Comment:
Env module is being removed in rack (rack/rack#1751). We might monkey patch or change rack's `Utils.default_query_parser`. (would need to check whether that applies to our current version)

Reply:
This one also did the job:
```
Rack::Utils.default_query_parser = Insights::API::Common::QueryParser.new(Rack::Utils.default_query_parser)
```

Reply:
@vkrizan I'm a bit concerned about how would this behave in code reloads, but I'll do some experiments with it

Reply:
Good point.

Reply:
It works, changed it :medal_military: thanks